brabbuss/refactor-tractor
# Feature/inputs and posts es

### What does this PR do?
1. @Estaffieri completed POST requests for all needed info #54 
2. Changes to script.js - refactoring and symantic changes
3. Bug fixes to get the correct information to display

### How should this be tested?
1. Test by opening console and submit new data under the actual web app UI - any errors will be caught and log an error message
2. All displays should update to reflect what the helper text for that data is indicating (i.e. all users ounces will show the average of all users ounces record)
